src/components/ReceiverInput.tsx

```diff
@@ -66,6 +66,7 @@ function ReceiverInput() {
           className={classes.address}
           fullWidth
           onChange={onReceiverChange}
+          autoComplete="off"
```

Comment:
Its better to allo the user have autocomplete - this way if he uses a specific address again and again - he does not have to copy/paste non-stop

Reply:
IMHO this will be better addressed with a custom address book. Having a bunch of random (past; one-time) addresses being displayed isn't that helpful. We have identicons to also quickly verify the correctness of address without going character by character.

Reply:
I agree with the address book. That would be the ideal solution, but for now i much prefer disabling the autocomplete because after a couple uses it gets really messy and annoying.